src/App/Documentation/components/Footer/index.js

```diff
@@ -1,12 +1,16 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
-import { ComponentPreview, DocContainer } from "#";
+import { ComponentPreview, DocContainer, Property } from "#";
 import FooterComponent from "@/Footer";
 
 const Overview = () => (
     <>
         <h2 id="overview">Overview</h2>
-        <p>Footer.</p>
+        <p>
+            Our footer component comes with the same brand color as our topbar.
+            It also supports aligning text with <Property value=".page-footer-rights"/> when a page reaches the small <Link to="/docs/core/breakpoints">breakpoint</Link>.
```

Comment:
Is the last sentence relevant?
If yes could it be easier explained, something like "`.page-footer-rights` is centered on mobile"?

Reply:
I agree. I'll change the wording.